### ISSUE TITLE
Cosmos series, such as how to generate celestia address

### ISSUE THREAD

**Issue by b1tboy** [ebellocchia/bip_utils#132]

**Comment by ebellocchia** [comment]
If it's not supported, I should add it

**Comment by b1tboy** [comment]
Or it can support expansion. Cosmos supports many chains, but their algorithms should be similar

![image](https://github.com/ebellocchia/bip_utils/assets/4327346/1be4155a-f3f1-4bc9-81cb-f4c6308e6bdc)

**Comment by ebellocchia** [comment]
Yes it's always the same, in fact you can already generate any Cosmos address by using BIP32
